src/lib.rs

```diff
@@ -152,16 +152,14 @@ pub mod constants;
 pub mod ecdh;
 pub mod ffi;
 pub mod key;
+#[cfg(feature = "std")]
```

Comment:
Needs to be `feature = "recovery"` :)

Reply:
oops, apparently forgot cleaning up after  removing the correct line and adding this wrong line for testing